iOSPhotoEditor/PhotoEditor+Controls.swift

```diff
@@ -114,16 +125,45 @@ extension PhotoEditorViewController {
             subview.removeFromSuperview()
         }
         
+        backupColoredLines = coloredLines
+        coloredLines = [coloredLine]()
+        removedLines = [coloredLine]()
+        canResetLines = undoManager?.canUndo == true
+        addControls(animated: false)
+        
         tracker?.track(event: .revert)
     }
     
+    @IBAction func resetButtonTapped(_ sender: Any) {
+        let alert = UIAlertController(title: "Möchten Sie wirklich alle Markierungen wiederherstellen?", message: nil, preferredStyle: .actionSheet)
+        alert.addAction(UIAlertAction(title: "Abbrechen", style: .destructive))
+        alert.addAction(UIAlertAction(title: "Wiederherstellen", style: .cancel) { _ in
+            self.reset()
+        })
+        present(alert, animated: true, completion: nil)
```

Comment:
Führt zum Crash auf dem iPad
_*** Terminating app due to uncaught exception 'NSGenericException', reason: 'Your application has presented a UIAlertController (<UIAlertController: 0x7f77bf052800>) of style UIAlertControllerStyleActionSheet from UINavigationController (<UINavigationController: 0x7f77df037600>). The modalPresentationStyle of a UIAlertController with this style is UIModalPresentationPopover. You must provide location information for this popover through the alert controller's popoverPresentationController. You must provide either a sourceView and sourceRect or a barButtonItem.  If this information is not known when you present the alert controller, you may provide it in the UIPopoverPresentationControllerDelegate method -prepareForPopoverPresentation.'_